src/main/io/dji_osd_symbols.h

```diff
@@ -71,6 +71,26 @@
 #define DJI_SYM_AH_DECORATION           0x13
 #define DJI_SYM_SMALL_CROSSHAIR         0x7E
 
+// Crosshair Styles
+#define DJI_SYM_AH_CH_TYPE3             0x00
+#define DJI_SYM_AH_CH_TYPE3_1           0x7E
+#define DJI_SYM_AH_CH_TYPE3_2           0x00
+#define DJI_SYM_AH_CH_TYPE4             0x2D
+#define DJI_SYM_AH_CH_TYPE4_1           0x7E
+#define DJI_SYM_AH_CH_TYPE4_2           0x2D
+#define DJI_SYM_AH_CH_TYPE5             0x17
+#define DJI_SYM_AH_CH_TYPE5_1           0x7E
+#define DJI_SYM_AH_CH_TYPE5_2           0x17
+#define DJI_SYM_AH_CH_TYPE6             0x00
+#define DJI_SYM_AH_CH_TYPE6_1           0x09
+#define DJI_SYM_AH_CH_TYPE6_2           0x00
+#define DJI_SYM_AH_CH_TYPE7             0x78
+#define DJI_SYM_AH_CH_TYPE7_1           0x7E
+#define DJI_SYM_AH_CH_TYPE7_2           0x77
+#define DJI_SYM_AH_CH_TYPE8             0x02
+#define DJI_SYM_AH_CH_TYPE8_1           0x7E
+#define DJI_SYM_AH_CH_TYPE8_2           0x03
+
```

Comment:
You shouldn't create these new defines. Because they are not actually symbols. Just use the existing definitions or characters to create the crosshairs in the other file. There shouldn't be multiple definitions for the same character reference. For example, only one instance of 0x7E.